pkg/cache/internal/informers.go

```diff
@@ -92,6 +93,25 @@ type tracker struct {
 	Metadata     map[schema.GroupVersionKind]*Cache
 }
 
+// GetOptions provides configuration to customize the behavior when
+// getting an informer.
+type GetOptions struct {
+	// DoNotBlockUntilSynced tells Get() to return the informer immediately,
+	// without waiting for its cache to sync.
+	DoNotBlockUntilSynced bool
```

Comment:
Lets avoid the negation to simplify this:
```suggestion
        // BlockUntilSynced controls if the informer retrieval will block until the informer is synced. Defaults to `true`
	BlockUntilSynced *bool
```

Reply:
Done